drivers/pds/lib/utils.go

```diff
@@ -291,31 +323,10 @@ func SetupPDSTest(ControlPlaneURL, ClusterType, AccountName string) (string, str
 	if len(clusterID) > 0 {
 		log.InfoD("clusterID %v", clusterID)
 	} else {
-		log.Fatalf("Cluster ID is empty")
+		log.InfoD("Cluster ID is empty")
```

Comment:
is it ok if clusterid is empty?

Reply:
I have updated code to return error telling cluster id is empty